ci.sh

```diff
@@ -195,6 +217,40 @@ if [ "$PROVIDER_NAME" = "coverage" ]; then
 fi
 
 echo "Build test"
+
+if [ "$PROVIDER_NAME" = "all" ]; then
```

Comment:
Could this be split into a different CI job? It seems to almost double the `all-providers` job to 17m, whereas the others finish in under 10 mins (not taking the Docker image build time into consideration, of course)

Reply:
I think this is a good idea but will need some more logic in the `ci.sh` script to be able to isolate that code with a new parameter. At the moment, CI execution time is not a big problem so I think this is fine creating an issue for this and doing it as part of another PR?

Reply:
Created #447